logging/src/main/java/org/jboss/as/logging/logging/LoggingLogger.java

```diff
@@ -926,4 +926,15 @@
     @LogMessage(level = ERROR)
     @Message(id = 88, value = "Could not determine %s had any children resources.")
     void errorDeterminingChildrenExist(@Cause Throwable cause, String childType);
+
+    /**
+     * Logs a warning message indicating the log manager does not appear to the {@link org.jboss.logmanager.LogManager}.
+     *
+     * @param logManagerName the log manager system property value
+     */
+    @LogMessage(level = WARN)
```

Comment:
Is this a WARN, not an INFO? The user set the property, which is unlikely to happen by mistake or without prior research, so they are presumably somewhat knowledgable.

In the past I had the mentality of "assume a WARN or ERROR log message means some poor sysadmin might get paged." I know that we aren't so restrictive everywhere though.